src/util/uniqueType.ml

```diff
@@ -0,0 +1,16 @@
+open GoblintCil
+
+(* Type invariant variables. *)
+let type_inv_tbl = Hashtbl.create 13
```

Comment:
Is it ok to not marshal this table?

Reply:
Possibly not, but `exp.unique` probably hasn't been used in the era of incremental and I wasn't going to try to refactor all this here.